packages/react-scripts/scripts/start-ssr.js

```diff
@@ -0,0 +1,62 @@
+// @remove-on-eject-begin
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+// @remove-on-eject-end
+'use strict';
+
+// Do this as the first thing so that any code reading it knows the right env.
+process.env.BABEL_ENV = 'development';
+process.env.NODE_ENV = 'development';
+
+// Makes the script crash on unhandled rejections instead of silently
+// ignoring them. In the future, promise rejections that are not handled will
+// terminate the Node.js process with a non-zero exit code.
+process.on('unhandledRejection', err => {
+  throw err;
+});
+
+// Ensure environment variables are read.
+require('../config/env');
+// @remove-on-eject-begin
+// Do the preflight check (only happens before eject).
+const verifyPackageTree = require('./utils/verifyPackageTree');
+if (process.env.SKIP_PREFLIGHT_CHECK !== 'true') {
+  verifyPackageTree();
+}
+const verifyTypeScriptSetup = require('./utils/verifyTypeScriptSetup');
+verifyTypeScriptSetup();
+// @remove-on-eject-end
+
+const fs = require('fs');
+const { createCompiler } = require('react-dev-utils/WebpackDevServerUtils');
+const webpack = require('webpack');
+
+const configFactory = require('../config/webpack.config.ssr');
+const paths = require('../config/paths');
+
+const statusFile = require('./utils/statusFile');
+
+const config = configFactory('development');
+const appName = require(paths.appPackageJson).name;
+const useYarn = fs.existsSync(paths.yarnLockFile);
+
+const compiler = createCompiler(webpack, config, appName, undefined, useYarn);
```

Comment:
Turns out the params of this function have changed to accept an object of these parameters so will update with the correct expected format for this

See here for example: https://github.com/Skyscanner/backpack-react-scripts/blob/9ab25eb68904847805d4a5681ab7e6a8620f0845/packages/react-scripts/scripts/start.js#L115